docs/source/tutorials/datalab/text.ipynb

```diff
@@ -62,7 +62,7 @@
     "You can use `pip` to install all packages required for this tutorial as follows:\n",
     "\n",
     "```ipython3\n",
-    "!pip install sklearn sentence-transformers\n",
+    "!pip install sklearn sentence-transformers datasets\n",
```

Comment:
```suggestion
    "!pip install sklearn sentence-transformers \n",
```

Don't need to explicitly state `datasets` as it will be installed with `cleanlab[datalab]`